Makefile

```diff
@@ -56,6 +56,28 @@ ifneq ($(TAG),latest)
 BUNDLE_VERSION ?= --version=$(shell sed s/^v// <<< $(TAG))
 endif
 
+# To get the GIT commit id
+ifndef GIT_COMMIT
+GIT_COMMIT=$(shell git rev-list -1 HEAD)
+endif
+
+# Fetch the git tag
+ifndef GIT_TAG
+GIT_TAG=$(shell git describe --tags --exact-match --abbrev=0 2>/dev/null)
+endif
+
+# In the above command, if it fails to fetch the git tag, then this
+# command will fetch the short commit hash
+ifeq ($(GIT_TAG),)
+GIT_TAG=$(shell git rev-parse --short HEAD)
```

Comment:
Consider using `git describe` to get a more user-friendly tag-based line:
```
$ git describe --tags HEAD
v0.6.0-56-g342d4570
```

Reply:
I tried git describe command, but it fails if I am on main branch,
```
[ kubernetes-csi-addons]# git describe --tags HEAD
fatal: No names found, cannot describe anything.
[ kubernetes-csi-addons]# git branch
* main
```

Reply:
can you try on  origin main not on your local main `git checkout origin/main`?

Reply:
It worked. Thanks.